### DISCUSSION TITLE
Please add the customized multi-page view function

### DISCUSSION THREAD

**Discussion by iqs2014** [sumatrapdfreader/sumatrapdf#3925] [Ideas]
Please add the customized multi-page view function in the View menu, because there is a 21:9 monitor now.
Thank you all very much!

**Comment by GitHubRulesOK** [comment]
it does not matter how tall your screen is you can only view one line at a time so fit width will work on portrait or landscape see here you can use 2 apps at the same time
![F400089166-FIL-global-001](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/7d03ee48-a12e-467a-ac8a-8b48489d687e)
For 5120 x 1440 wide screen just ensure you are using 2 pages side by side 
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/b4aa20c9-4c7e-4fc9-b5de-94d7e9bdbd70)

**Reply by iqs2014** [reply]
thanks，I think if the software can customize >2 pages to multi-page view to give user more.
like this pic, I hope it can customize 4 pages to multi-page view.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/152386974/6f0943ed-1605-410d-8276-5d2b5402d417)

**Reply by GitHubRulesOK** [reply]
I have 2 eyes so only need one page at a time a maximum of 2 if a diagram is "spread" there is no logic in showing horizontally
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/50947339-997f-4033-9b57-bae42ba00a1b)

Here Adobe Acrobat can show smaller pages to fit a screen width (several other readers can do that) but its not readable without a taller monitor.
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/95ffafd7-da04-4f0d-adc8-054d35789c8f)

And Foxit has twin views for 4 pages, so closest to a 4 up system. (Oddly it cannot display all Chinese GB fonts)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/4121c070-0494-488e-8dbc-8e6b1dd4bf8c)
xpdf
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/6cd89ec0-9174-4b10-94ad-f2a7ae93c72a)

And if doing a full touch screen or using tablet even SumatraPDF can work multiple pages in Portrait mode (not recommended)
![image](https://github.com/sumatrapdfreader/sumatrapdf/assets/12754913/17e4211b-122b-49a4-a1b8-e78c9eb2ddbf)

**Reply by iqs2014** [reply]
thanks again,
Yes, your reply is my needs.
This is useful for us to think when reading an article, so that we can read the information on one screen.
what software is it? 
got it, thanks again, hope you can consider my advice.

**Reply by iqs2014** [reply]
like the pic of word, only multi-page to adjust.
![bc5cacdb8df92318bf742fce4688d9b](https://github.com/sumatrapdfreader/sumatrapdf/assets/152386974/45bb4462-8df1-4a7f-9985-9cc09e485ced)
![9ab1a299ca7116cf411b24ea4c5a762](https://github.com/sumatrapdfreader/sumatrapdf/assets/152386974/d196df79-db98-4a89-b204-38aee644af8d)
